clap_complete/tests/testsuite/fish.rs

```diff
@@ -142,6 +142,7 @@ fn register_completion() {
 
 #[test]
 #[cfg(unix)]
+#[ignore = "Could not set up terminal, TERM environment variable not set"]
```

Comment:
These `*complete` tests are already broken before, CI couldn't run them because missing fish, zsh, etc.

Reply:
Maybe we should have a tracking issue for this.